src/index.js

```diff
@@ -96,6 +102,15 @@ async function monitor(options) {
   }
   await auditlog.setup();
 
+  // read gitVersionFile, if gitVersion is not set
+  let gitVersion;
+  if (!options.gitVersion) {
+    let gitVersionFile = path.resolve(rootdir.get(), options.gitVersionFile);
+    if (fs.existsSync(gitVersionFile)) {
```

Comment:
I generally like to open files with a try catch rather than if-then just because of bad experiences with race conditions, but I'm happy either way.

Reply:
fixed